edml-1.0.0.json

```diff
@@ -0,0 +1,215 @@
+              "enum": [
+                "NULL",
+                "ABORT"
+              ]
+            },
+            "destinationName": {
+              "type": "string",
+              "description": "Name of the Exasol column. required if used at root level. Else the name of the DynamoDB property is used."
+            },
+            "description": {
+              "$ref": "#/definitions/description"
+            },
+            "key": {
+              "$ref": "#/definitions/key"
+            }
+          },
+          "additionalProperties": false
+        },
+        "toDecimalMapping": {
+          "title": "toDecimal mapping definition",
+          "description": "Maps the selected DynamoDB property to an Exasol DECIMAL column.\n\n*Supported DynamoDB types:* Numbers (N).",
+          "properties": {
+            "decimalPrecision": {
+              "type": "integer",
+              "description": "Precision of the Exasol DECIMAL type. See https://docs.exasol.com/sql_references/data_types/datatypedetails.htm\n\ndefault = 18",
+              "default": 18
+            },
+            "decimalScale": {
+              "type": "integer",
+              "description": "Scale of the Exasol DECIMAL type. See https://docs.exasol.com/sql_references/data_types/datatypedetails.htm\n\ndefault = 0",
+              "default": 0
+            },
+            "overflowBehaviour": {
+              "description": "Behaviour to apply in case a value exceeds the size of the DECIMAL column.\n\nNULL: use null instead ABORT: abort whole query",
+              "default": "ABORT",
+              "enum": [
+                "NULL",
+                "ABORT"
+              ]
+            },
+            "notNumericBehaviour": {
+              "description": "Behaviour to apply in case a value is not a number\n\nNULL: use null instead ABORT: abort whole query",
+              "default": "ABORT",
+              "enum": [
+                "NULL",
+                "ABORT"
+              ]
+            },
+            "destinationName": {
+              "type": "string",
+              "description": "Name of the Exasol column.\n\nDefault: name of the DynamoDB property is used."
+            },
+            "description": {
+              "$ref": "#/definitions/description"
+            },
+            "key": {
+              "$ref": "#/definitions/key"
+            }
+          },
+          "additionalProperties": false
+        },
+        "toTableMapping": {
+          "title": "toTable mapping definition",
+          "description": "This mapping is used for normalizing documents from DynamoDB. It creates a new table in the resulting virtual schema named: \\<NAME_OF_PARENT\\>_\\<GIVEN_NAME\\>.\n\n The columns of that new table are defined using the mapping property. If the parent table defined a key using one or more toString mappings for a DynamoDB primary key or secondary index this will be used as foreign key.\n\n*Supported DynamoDB types:* List (L).",
```

Comment:
```suggestion
          "description": "This mapping is used for normalizing documents from DynamoDB. It creates a new table in the resulting virtual schema named: \\<NAME_OF_PARENT\\>_\\<GIVEN_NAME\\>.\n\n The columns of that new table are defined using the mapping property. If the parent table defines a key using one or more toString mappings for a DynamoDB primary key or secondary index, this will be used as foreign key.\n\n*Supported DynamoDB types:* List (L).",
```

Reply:
```suggestion
          "description": "This mapping is used for normalizing documents from DynamoDB. It creates a new table in the resulting virtual schema named: \\<NAME_OF_PARENT\\>_\\<GIVEN_NAME\\>.\n\n The columns of that new table are defined using the mapping property. If the parent table defines a key using one or more toVarcharMappings for a DynamoDB primary key or secondary index, this will be used as foreign key.\n\n*Supported DynamoDB types:* List (L).",
```